model.go

```diff
@@ -10,3 +10,18 @@ type Paging struct {
 	PerPage int64 `json:"perPage"`
 	Offset  int64 `json:"offset"`
 }
+
+type UpdateSearchConcurrencySettingsScheduleReq struct {
+	MaxSearchesPer *int
+	AutoSummaryPer *int
```

Comment:
Is there a reason why these are pointers?  Having these as pointers (probably) unnecessarily puts them on the heap, and an int is the same size as a pointer.

Reply:
On second glace, I see it's because you want them to be optional.  In this case, I'm fine with it actually.

Reply:
Also, could we add comments for each parameter equating to the "Description" from the splunk docs?  As well as adding the other parameters: `max_searches_per_cpu` `base_max_searches` `max_rt_search_multiplier`

Reply:
It was because they are optional and 0 is a value. But like you want if you want a value fine for me to change

Reply:
![Screenshot from 2020-10-15 19-41-59](https://user-images.githubusercontent.com/22316360/96172552-87665c00-0f1e-11eb-9e34-72061da63cce.png)

The Post request has just max_searches_perc and auto_summary_perc

Reply:
is the documentation wrong @vertoforce ?

Reply:
Oh my bad I must have been looking at the wrong endpoint!